doc/usecases/specification/dockerd.spec

```
@@ -0,0 +1,446 @@
+
+[dns/opt/_]
+meta:/type = string
+meta:/description = DNS options to use
+
+[dns/search/_]
+meta:/type = string
+meta:/description = DNS search domains to use
+
+[exec/opt/_]
+meta:/type = string
+meta:/description = Runtime execution options
+
+[exec/root/_]
+meta:/type = string
+meta:/description = Root directory for execution state files
+meta:/default = /var/run/docker
+
+[experimental]
+meta:/type = boolean
+meta:/description = Enable experimental features
+meta:/default = false
+
+[fixed-cidr]
+meta:/type = string
+meta:/description = IPv4 subnet for fixed IPs
+
+[fixed-cidr-v6]
+meta:/type = string
+meta:/description = IPv6 subnet for fixed IPs
+
+[group]
+meta:/type = string
+meta:/description = Group for the unix socket
+meta:/default = docker
+
+[host/_]
+meta:/type = string
+meta:/description = Daemon socket(s) to connect to
+
+[host-gateway-ip]
+meta:/enum/ipaddr = ipv4
+meta:/description = IP address that the special 'host-gateway' string in add-host resolves to. Defaults to the IP address of the default bridge.
+
+[http-proxy]
+meta:/type = string
+meta:/description = HTTP proxy URL to use for outgoing traffic
+
+[https-proxy]
+meta:/type = string
+meta:/description = HTTPS proxy URL to use for outgoing traffic
+
+[icc]
+meta:/type = boolean
+meta:/description = Enable inter-container communication
+meta:/default = true
+
+[init]
+meta:/type = boolean
+meta:/description = Run an init in the container to forward signals and reap processes
+meta:/default = false
+
+[init-path]
+meta:/type = string
+meta:/description = Path to the docker-init binary
+
+[insecure-registries/#]
+meta:/array/min = 0
+meta:/description = Enable insecure registry communication by specifying the insecure-registries.
+
+[ip]
+meta:/description = Default IP when binding container ports
+meta:/enum/ipaddr = ipv4
+meta:/default = 0.0.0.0
+
+[ip-forward]
+meta:/type = boolean
+meta:/description = Enable net.ipv4.ip_forward
+meta:/default = true
+
+[ip-masq]
+meta:/type = boolean
+meta:/description = Enable IP masquerading
+meta:/default = true
+
+[iptables]
+meta:/type = boolean
+meta:/description = Enable addition of ip6tables rules
+meta:/default = false
+
+[iptables]
+meta:/type = boolean
+meta:/description = Enable addition of iptables rules
+meta:/default = true
+
+[ipv6]
+meta:/type = boolean
+meta:/description = Enable IPv6 networking
+meta:/default = false
+
+[labels/_]
+meta:/type = string
+meta:/description = Set key=value labels to the daemon
+
+[live-restore]
+meta:/type = boolean
+meta:/description = Enable live restore of docker when containers are still running
+meta:/default = false
+
+[log/driver]
+meta:/type = boolean
+meta:/description = Default driver for container logs
+meta:/default = json-file
+
+[log/level]
+meta:/description = Set the logging level
+meta:/enum/check = #4
+meta:/enum/check/#0 = debug
+meta:/enum/check/#1 = info
+meta:/enum/check/#2 = warn
+meta:/enum/check/#3 = error
+meta:/enum/check/#1 = fatal
```

```suggestion
meta:/enum/check/#4 = fatal
```
I guess it should be 4, no?